### PR TITLE
Add hints functionality and a fifty-fifty hint

### PR DESCRIPTION
**Pull request by patrickteivas**
This PR:
* Sorts players by score on scoreboard
* Adds hints functionality and a fifty-fifty hint
* Adds `.cache` and `dist` folders to `.gitignore`
* Removes `.cache` and `dist` folders from current commit
<br>
Server-side code edits Kasparsu/triviaserver/pull/3

### CODE REVIEW

File: src/Play.vue [hunk 1]
```diff
@@ -64,6 +82,26 @@
...
+
+                } else {
+
+                }
+            }
+        },
+        computed: {
+            availableHintsCount() {
+                let count = 0;
+                for (const hint in this.hints) {
```
Comment (Kasparsu) [inline src/Play.vue]: reduce maybe? But is okay as well

File: src/Play.vue [hunk 2]
```diff
@@ -64,6 +82,26 @@
             answer(option){
                 this.ws.send(JSON.stringify({sender:'player', id: this.uid, action:'answer', data: {answer: option}}));
                 this.options = [];
+            },
+            getHint(hint) {
+                if(hint.available) {
+                    this.ws.send(JSON.stringify({sender:'player', id: this.uid, action:'getHint', data: {type: hint.type}}));
+                    hint.available = false;
+
+                } else {
```
Comment (Kasparsu) [inline src/Play.vue]: else????

Comment (patrickteivas) [reply src/Play.vue]: Resolved by 649f85f
I also added a comment with an alternative solution.

File: src/Play.vue (hunk 1 above)
Comment (patrickteivas) [reply src/Play.vue]: Resolved by 649f85f
I also added a comment with an alternative solution.

File: src/Play.vue (hunk 2 above)
Comment (patrickteivas) [reply src/Play.vue]: Resolved by 0a92335

**Pull request closed without merging**